src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
```

Comment:
All fields on this object and the other CMDT object need to have an InlineHelpText attribute that explains how the field is used. If additional (technical) details are relevant, that can go into the Description attribute of the field.

Reply:
Thanks for adding the object descriptions. Each individual field on both of the CMDT objects also require **both** a description and inlineHelpText attribute.